config/routes.rb

```diff
@@ -0,0 +1,102 @@
+Rails.application.routes.draw do
+  resources :movies
+# get "/movies" => "movies#index", as: :movies
+# post "/movies" => "movies#create"
+# get "/movies/:id" => "movies#show", as: :movie
+# patch "/movies/:id" => "movies#update"
+# put "/movies/:id" => "movies#upate"
+# delete "/movies/:id" => "movies#destroy"
+# get "/movies/new" => "movies#new", as: :new_movie
+# get "/movies/:id/edit" => "movies#edit", as: :edit_movie
+  patch "movies/:id/upvote" => "movies#upvote", as: :movies_upvote
+
+  resources :books
+# get "/books" => "books#index", as: :books
+# post "/books" => "books#create"
+# get "/books/:id" => "books#show", as: :book
+# patch "/books/:id" => "books#update"
+# put "/books/:id" => "books#upate"
+# delete "/books/:id" => "books#destroy"
+# get "/books/new" => "books#new", as: :new_book
+# get "/books/:id/edit" => "books#edit", as: :edit_book
+  patch "books/:id/upvote" => "books#upvote", as: :books_upvote
+
+  resources :albums
+  # get "/albums" => "albums#index", as: :albums
+  # post "/albums" => "albums#create"
+  # get "/albums/:id" => "albums#show", as: :album
+  # patch "/albums/:id" => "albums#update"
+  # put "/albums/:id" => "albums#upate"
+  # delete "/albums/:id" => "albums#destroy"
+  # get "/albums/new" => "albums#new", as: :new_albym
+  # get "/albums/:id/edit" => "albums#edit", as: :edit_album
+  patch "albums/:id/upvote" => "albums#upvote", as: :albums_upvote
+  # I was orignally going to not include all of the resource functionality. Yet, because I know another wave is coming with potential web auth. I have chosen to include this.
+  resources :users
```

Comment:
I appreciate the comment explaining the inclusion, and it's a reasonable thought (we certainly did ask y'all to go back and update a project to add authentication!).

But this falls under the You're Not Gonna Need It (YAGNI) rule which basically says "don't write code for features that you're not yet implementing".